package.json

```diff
@@ -9,7 +9,7 @@
     "start": "node dist/src/index.js",
     "tsoa": "tsoa spec-and-routes",
     "gen-client": "npm run gen-client-backoffice && npm run gen-client-client",
-    "gen-client-backoffice": "openapi-ts --input ./build/swagger.json --output ../narrowcasting-backoffice/src/api/",
+    "gen-client-backoffice": "openapi-ts -o ../aurora-backoffice/src/api/",
     "gen-client-client": "openapi-ts --input ./build/swagger.json --output ../narrowcasting-client/src/api/",
```

Comment:
I have updated the `gen-client-client` and `gen-client-backoffice` commands. However, since the new `hey-api` version is installed now, and has breaking changes compared to the previous version, the `gen-client-client` will generate an incompatible client now.

I shall open a PR on the client as well where I use the new client there as well.

Reply:
See https://github.com/GEWIS/aurora-client/pull/2